docs/content/tutorials/cluster.md

```diff
@@ -0,0 +1,327 @@
+being tuned to the hardware they run on. If you are using [r3.2xlarge](https://aws.amazon.com/ec2/instance-types/#r3) 
+EC2 instances, or similar hardware, the configuration in the distribution is a 
+reasonable starting point.
+
+If you are using different hardware, we recommend adjusting configurations for your specific 
+hardware. The most commonly adjusted configurations are:
+
+- `-Xmx` and `-Xms`
+- `druid.server.http.numThreads`
+- `druid.processing.buffer.sizeBytes`
+- `druid.processing.numThreads`
+- `druid.query.groupBy.maxIntermediateRows`
+- `druid.query.groupBy.maxResults`
+- `druid.server.maxSize` and `druid.segmentCache.locations` on Historical Nodes
+- `druid.worker.capacity` on MiddleManagers
+
+```note
+Keep -XX:MaxDirectMemory >= numThreads*sizeBytes, otherwise Druid will fail to start up..
+```
+
+Please see the Druid [configuration documentation](../configuration/index.html) for a full description of all 
+possible configuration options.
+
+## Tune Druid Brokers
+
+Druid Brokers also benefit greatly from being tuned to the hardware they  
+run on. If you are using [r3.2xlarge](https://aws.amazon.com/ec2/instance-types/#r3) EC2 instances, 
+or similar hardware, the configuration in the distribution is a reasonable starting point.
+
+If you are using different hardware, we recommend adjusting configurations for your specific 
+hardware. The most commonly adjusted configurations are:
+
+- `-Xmx` and `-Xms`
+- `druid.server.http.numThreads`
+- `druid.cache.sizeInBytes`
+- `druid.processing.buffer.sizeBytes`
+- `druid.processing.numThreads`
+- `druid.query.groupBy.maxIntermediateRows`
+- `druid.query.groupBy.maxResults`
+
+```note-caution
+Keep -XX:MaxDirectMemory >= numThreads*sizeBytes, otherwise Druid will fail to start up..
+```
+
+Please see the Druid [configuration documentation](../configuration/index.html) for a full description of all 
+possible configuration options.
+
+## Start Coordinator, Overlord, Zookeeper, and metadata store
+
+Copy the Druid distribution and your edited configurations to your coordination 
+server. If you have been editing the configurations on your local machine, you can use *rsync* to 
+copy them:
+
+```bash
+rsync -az druid-0.9.0/ COORDINATION_SERVER:druid-0.9.0/
+```
+
+Log on to your coordination server and install Zookeeper:
+ 
+```bash
+curl http://www.gtlib.gatech.edu/pub/apache/zookeeper/zookeeper-3.4.6/zookeeper-3.4.6.tar.gz -o $zookeeper-3.4.6.tar.gz
+tar xzf $zookeeper-3.4.6.tar.gz
+cd zookeeper-3.4.6
+cp conf/zoo_sample.cfg conf/zoo.cfg
+./bin/zkServer.sh start
+
+```note-caution
+In production, we also recommend running a ZooKeeper cluster on its own dedicated hardware.
+```
+
+On your coordination server, *cd* into the distribution and start up the coordination services:
+
+```bash
+java `cat conf/druid/coordinator/jvm.config | xargs` -cp conf/druid/_common:conf/druid/coordinator:lib/*.jar io.druid.cli.Main server coordinator > var/log/druid/coordinator.log
```

Comment:
I guess for including all jars in the `lib` folder instead of `lib/*.jar` it should be just `lib/*`